core/src/components/alert/alert.scss

```diff
@@ -54,6 +54,7 @@
 
 .alert-wrapper {
   display: flex;
+  position: relative;
```

Comment:
This was added to resolve an issue where inline alerts show underneath the backdrop, due to the added `ion-delegate-host` wrapper messing up the stacking.

Reply:
Hmm this looks like a bug. `ion-delegate-host` should only be added to `ion-modal` and `ion-popover` since they accept user components.

Reply:
`ion-delegate-host` doesn't have any style responsibility. It was just used to track the existing wrapper element that was created, so we didn't create another one. e.g. During the PR at one point I was setting a private flag vs. adding a class. We landed on a class just for simplicity and to avoid someone accidentally setting that flag. 

I would need more information to what caused this style being introduced.